spec/dry_run_spec.rb

```diff
@@ -1,17 +1,11 @@
 require 'spec_helper'
 
 describe Slackistrano do
-  before(:each) do
-    Rake::Task['load:defaults'].execute
-  end
-
-  %w[updating reverting updated reverted failed].each do |stage|
+  %w(updating reverting updated reverted failed).each do |stage|
     it "does not post to slack on slack:deploy:#{stage}" do
-      set "slack_run_#{stage}".to_sym, ->{ true }
-      expect(Capistrano::Configuration.env.send(:config)).to receive(:[]).with(:sshkit_backend).and_return(SSHKit::Backend::Printer)
+      set "slack_run_#{stage}".to_sym, -> { true }
```

Comment:
Please add this line:
allow(Net::HTTP).to receive(:post_form).and_return double(code: '200' )